### PR TITLE
addproductcommandparser updated (bug in prev commit fixed), and clear…

### PR DESCRIPTION
**Pull request by dyeosy98**
…distributors command added. clear should now be clearproducts

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/ClearCommand.java [hunk 1]
```diff
@@ -12,7 +12,7 @@
  */
 public class ClearCommand extends Command {
 
-    public static final String COMMAND_WORD = "clear";
+    public static final String COMMAND_WORD = "clearproducts";
     public static final String MESSAGE_SUCCESS = "Address book has been cleared!";
```
Comment (amirulmaricar) [inline src/main/java/seedu/address/logic/commands/ClearCommand.java]: "**Inventory** has been cleared!" 

What do you think?

File: src/main/java/seedu/address/model/distributor/Distributor.java [hunk 2]
```diff
@@ -95,7 +95,7 @@ public String toString() {
         builder.append(getDistName())
```
Comment (amirulmaricar) [inline src/main/java/seedu/address/model/distributor/Distributor.java]: I dont' see .append( " Name: ")

File: src/main/java/seedu/address/logic/commands/ClearCommand.java (hunk 1 above)
Comment (dyeosy98) [reply src/main/java/seedu/address/logic/commands/ClearCommand.java]: Yes I think that should be the output message!

File: src/main/java/seedu/address/model/distributor/Distributor.java (hunk 2 above)
Comment (dyeosy98) [reply src/main/java/seedu/address/model/distributor/Distributor.java]: Thanks Amirul! Editing

**Pull request merged**